docs/scoring.md

```diff
@@ -2,9 +2,9 @@
 
 The goal of this document is to explain how scoring works in Lighthouse and what to do to improve your Lighthouse scores.
 
-If you want a more comprehensive spreadsheet of this document to understand weighting and scoring, check out the [scoring spreadsheet](https://docs.google.com/spreadsheets/d/1up5rxd4EMCoMaxH8cppcK1x76n6HLx0e7jxb0e0FXvc):
+If you want a more comprehensive version of this document to understand weighting and scoring, check out the [scoring calculator](https://paulirish.github.io/lh-scorecalc/):
```

Comment:
@paulirish seemed fine to do this switch now? Even though more stuff below needs to be updated

Reply:
> @paulirish seemed fine to do this switch now? Even though more stuff below needs to be updated

oh, and I just found #10676 :/